client/src/components/MobileMenuToggle/MobileMenuToggle.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Component } from 'react';
+
+class MobileMenuToggle extends Component {
+
+  constructor() {
+    super();
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    e.preventDefault();
+
+    if (typeof this.props.onClick === 'function') {
+      this.props.onClick(e);
+    }
+  }
+
+  render() {
+    const classes = ['cms-mobile-menu-toggle'];
+    if (this.props.isOpen) {
+      classes.push('cms-mobile-menu-toggle--open');
+    }
+
```

Comment:
This would be a good opportunity to use the `classnames` library, which we already have installed. It's a much more declarative way to assign classes conditionally.

```
const classes = classnames({
  'cms-mobile-menu-toggle': true,
  'cms-mobile-menu-toggle--open': this.props.isOpen
});
```